docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md

```diff
@@ -289,5 +289,39 @@ considerations apply:
 - If there is more than one Runtime Extension registered for the same Runtime Hook and at least one of them fails,
   all the registered Runtime Extension will be retried. See [Idempotence](#idempotence)
 
-Additional considerations about errors that apply only to a specific Runtime Hook will be documented in the hook-specific 
+Additional considerations about errors that apply only to a specific Runtime Hook will be documented in the hook-specific
 implementation documentation.
+
+## Tips & tricks
+
+After you implemented and deployed a Runtime Extension you can manually test it by sending HTTP requests.
+This can be for example done via kubectl:
+
+Via `kubectl create --raw`:
+
+```bash
+# Send a Discovery Request to the webhook-service in namespace default with protocol https on port 443:
+kubectl create --raw '/api/v1/namespaces/default/services/https:webhook-service:443/proxy/hooks.runtime.cluster.x-k8s.io/v1alpha1/discovery' \
+  -f <(echo '{"apiVersion":"hooks.runtime.cluster.x-k8s.io/v1alpha1","kind":"DiscoveryRequest"}') | jq
+```
+
+Via `kubectl proxy` and `curl`:
+
+```bash
+# Open a proxy with kubectl and then use curl to send the request
+## First terminal:
+kubectl proxy
+## Second terminal:
+curl -X 'POST' 'http://127.0.0.1:8001/api/v1/namespaces/default/services/https:webhook-service:443/proxy/hooks.runtime.cluster.x-k8s.io/v1alpha1/discovery' \
+  -d '{"apiVersion":"hooks.runtime.cluster.x-k8s.io/v1alpha1","kind":"DiscoveryRequest"}' | jq
+```
+
+For more details about the API of the Runtime Extensions please see <button onclick="openSwaggerUI()">Swagger UI</button>.
+
```

Comment:
How about adding a section that reads something like "For more details on proxy support please see [Proxies in Kubernetes](https://kubernetes.io/docs/concepts/cluster-administration/proxies/)".

The link would be useful as it will set context and provide guidance as necessary.

Reply:
Sounds good, feel free to open a PR to add this sentence

Reply:
PR: https://github.com/kubernetes-sigs/cluster-api/pull/7285

Reply:
Thx!